commands/commands_test.go

```diff
@@ -1,11 +1,19 @@
 package commands
 
 import (
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
+	"github.com/github/hub/console"
 )
 
+func TestMain(m *testing.M) {
+	console.Default = console.Console{Stdout: ioutil.Discard, Stderr: ioutil.Discard}
```

Comment:
Since TestMain affects all tests everywhere, should we put it in a file where it's more apparent that this is a global setting?

Also, shouldn't we capture the output instead of discard it?